mssqlcli/mssqlcliclient.py

```diff
@@ -335,19 +342,19 @@ def reset_connection_and_clients(sql_tools_client, *mssqlcliclients):
     try:
         sql_tools_client.shutdown()
         new_tools_client = SqlToolsClient()
```

Comment:
A client should not reinitialize the sqltoolsservice process. Can we move this a reset method in the sqltoolsclient client?